src/pages/PlayRoom.jsx

```diff
@@ -68,6 +70,32 @@ export function PlayRoom(props) {
         created: firebase.firestore.FieldValue.serverTimestamp(),
       });
   };
+
+  function shuffleArray(array) {
+    for (var i = array.length - 1; i > 0; i--) {
+      var j = Math.floor(Math.random() * (i + 1));
+      var temp = array[i];
+      array[i] = array[j];
+      array[j] = temp;
+    }
+  }
+  const colorMap = {};
+  const assignColor = (uid) => {
+    const arrColor = [
+      '#0055ff',
+      '#d64d0d',
+      '#cc0cb6',
+      '#0bad09',
+      '#f5e507',
+    ];
+    shuffleArray(arrColor);
+    if (!colorMap[uid]) {
+      colorMap[uid] =
+        arrColor[Object.entries(colorMap).length % arrColor.length];
+    }
+    return colorMap[uid];
+  };
+
```

Comment:
Add all this code in a separated utils.js file. Like the one at line 4: `import { textDescriptor } from '../../utils';`